runtime/Cpp/runtime/src/misc/Interval.h

```diff
@@ -28,7 +28,9 @@ namespace misc {
     Interval();
     explicit Interval(size_t a_, size_t b_); // For unsigned -> signed mappings.
     Interval(ssize_t a_, ssize_t b_);
-    virtual ~Interval() {};
+    Interval(Interval const&) = default;
+    virtual ~Interval();
```

Comment:
I think I'm missing something here ... The function after the destructor is the assignment operator.